src/factories/MetadataTagFactory.ts

```diff
@@ -171,13 +171,9 @@ const PARSER: Record<
     ----------------------------------------------------------- */
     format: (identifier, metadata, value, output) => {
         validate(identifier, metadata, output, "format", "string", ["pattern"]);
-        if (FORMATS.has(value) === false)
```

Comment:
Since the value can be `arbitrary` from now on, this check is not needed anymore